src/main/java/com/znsio/rpi/properties/ReportPortalPropertiesOverloader.java

```diff
@@ -0,0 +1,169 @@
+package com.znsio.rpi.properties;
+
+import com.znsio.rpi.utils.commandline.CommandLineExecutor;
+import com.znsio.rpi.utils.commandline.CommandLineResponse;
+import com.epam.reportportal.listeners.ListenerParameters;
+import com.epam.reportportal.utils.properties.PropertiesLoader;
+import com.epam.ta.reportportal.ws.model.attribute.ItemAttributesRQ;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.log4j.Logger;
+import org.testng.util.Strings;
+import org.testng.xml.Parser;
+import org.testng.xml.XmlSuite;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.*;
+
+import static com.znsio.rpi.utils.OverriddenVariable.getOverriddenStringValue;
+
+public class ReportPortalPropertiesOverloader {
+    private static ListenerParameters parameters = new ListenerParameters(PropertiesLoader.load());
+    private static Set<ItemAttributesRQ> itemAttributesRQSet = new HashSet<>();
+    private static final Logger LOGGER = Logger.getLogger(ReportPortalPropertiesOverloader.class.getName());
+    private static final String WEB_AUTOMATION = "WebAutomation";
+    private static final String APP_AUTOMATION = "AppAutomation";
+    private static final String NOT_SET = "not-set";
+    private static final String RP_PREFIX = "RP_";
+    private static final Properties config = Config.loadProperties(System.getProperty("CONFIG"));
+    private static final int DEFAULT_THREAD_COUNT = 1;
+
+    public static ListenerParameters getProperties() {
```

Comment:
does this need to be public?

Reply:
Yes, this is being called from ReportPortalListener class and they are not in the same package